### ISSUE TITLE
make simpleNetwork an alias of forceNetwork to reduce development efforts

### ISSUE THREAD

**Issue by cjyetman** [christophergandrud/networkD3#170]
I propose making the `simpleNetwork()` function an alias of the `forceNetwork()` function to reduce the effort of maintaining two very similar functions separately. I have a dev branch with this mostly implemented here: https://github.com/cjyetman/networkD3/tree/simpleNetwork_alias

one can install it for testing with...
```r
devtools::install_github('cjyetman/networkD3', ref = 'simpleNetwork_alias')
```

There is still some issue with multiplied opacity in the dev version `simpleNetwork()`, but that should be fixable. I could, probably should, also add `...` to the `simpleNetwork()` arguments so that one could pass down arguments to the underlying `forceNetwork()` function.

The only really difficult thing to replicate from `simpleNetwork()` to `forceNetwork()` is this changing color and locking a node into place on click. Is that an important feature to have, or would it be ok to leave that to the past?

The `simpleNetwork()` function in this dev branch currently looks like...
```r
simpleNetwork <- function(Data,
                          Source = 1,
                          Target = 2,
                          height = NULL,
                          width = NULL,
                          linkDistance = 50,
                          charge = -200,
                          fontSize = 7,
                          fontFamily = "serif",
                          linkColour = "#666",
                          nodeColour = "#3182bd",
                          nodeClickColour = "#E34A33",
                          textColour = "#3182bd",
                          opacity = 0.6,
                          zoom = F)
{
    # validate input
    if (!is.data.frame(Data))
        stop("data must be a data frame class object.")

    sources <- Data[[Source]]
    targets <- Data[[Target]]

    # Check if data is zero indexed
    check_zero(sources, targets)

    # create nodes data
    node_names <- factor(sort(unique(c(as.character(sources), as.character(targets)))))
    nodes <- data.frame(name = node_names, group = 1, size = 8)

    # create links data
    links <- data.frame(source = match(sources, node_names) - 1,
                        target = match(targets, node_names) - 1,
                        value = 1)

    # create options
    options = list(
        Links = links,
        Nodes = nodes,
        Source = 'source',
        Target = 'target',
        Value = 'value',
        NodeID = 'name',
        Group = 'group',
        linkDistance = linkDistance,
        charge = charge,
        fontSize = fontSize,
        fontFamily = fontFamily,
        linkColour = linkColour,
        colourScale = JS(paste0("d3.scaleOrdinal(['", nodeColour, "'])")),
        # nodeClickColour = nodeClickColour,
        # textColour = textColour,
        opacity = opacity,
        zoom = zoom,
        radiusCalculation = JS("d.nodesize"),
        Nodesize = 'size',
        linkWidth = "'1.5px'.toString()",
        opacityNoHover = 1
    )

    do.call(forceNetwork, options)
}
```

**Comment by christophergandrud** [comment]
I fully support this.

I don't think locking the colour on click is important, especially with the link highlight feature you're developing.

**Comment by cjyetman** [comment]
I removed the `nodeClickColour` and `textColour` arguments that are not compatible with forceNetwork(). I think this could be ready to merge if there are no issues. Getting both force network plots onto one JavaScript base is gonna make adding the other improvements (matching link color, directional arrows, etc.) a lot more manageable. Latest dev version can be installed with...
```r
devtools::install_github('cjyetman/networkD3', ref = 'simpleNetwork_alias')
```

**Comment by christophergandrud** [comment]
Looks great. Feel free to merge in.

**Comment by cjyetman** [comment]
This has been resolved by PR #181... closing
